src/components/dialogmoteunntak/DialogmoteunntakSkjema.tsx

```diff
@@ -10,34 +8,67 @@ import {
   CreateUnntakDTO,
   UnntakArsak,
 } from "@/data/dialogmotekandidat/types/dialogmoteunntakTypes";
-import DialogmoteunntakSkjemaArsakVelger, {
-  DialogmoteunntakSkjemaArsakVelgerFieldName,
-} from "@/components/dialogmoteunntak/DialogmoteunntakSkjemaArsakVelger";
-import DialogmoteunntakSkjemaBeskrivelse, {
-  dialogmoteunntakSkjemaBeskrivelseFieldName,
-  dialogmoteunntakSkjemaBeskrivelseMaxLength,
-} from "@/components/dialogmoteunntak/DialogmoteunntakSkjemaBeskrivelse";
-import { FlexGapSize, FlexRow } from "@/components/Layout";
 import { useValgtPersonident } from "@/hooks/useValgtBruker";
 import { useSettDialogmoteunntak } from "@/data/dialogmotekandidat/useSettDialogmoteunntak";
 import { SkjemaInnsendingFeil } from "@/components/SkjemaInnsendingFeil";
-import { validerTekst } from "@/utils/valideringUtils";
-import { Flatknapp, Hovedknapp } from "nav-frontend-knapper";
+import { SubmitHandler, useForm } from "react-hook-form";
+import { Button, Radio, RadioGroup, Textarea } from "@navikt/ds-react";
+import DialogmoteunntakSkjemaStatistikk from "@/components/dialogmoteunntak/DialogmoteunntakSkjemaStatistikk";
 
 export const texts = {
   noBrev: "Det blir ikke sendt ut brev ved unntak.",
   infoKandidatlist: `Når du setter unntak fra dialogmøte vil arbeidstakeren bli fjernet fra kandidatlisten. Dersom du på et senere tidspunkt vurderer at det likevel er nødvendig med et dialogmøte, kan du kalle inn til dialogmøte ved å søke deg frem til denne arbeidstakeren.`,
+  arsakLegend: "Årsak til unntak (obligatorisk)",
+  arsakErrorMessage: "Vennligst angi årsak.",
+  beskrivelseLabel: "Beskrivelse (valgfri)",
   send: "Sett unntak",
   avbryt: "Avbryt",
 };
 
-const StyledPanel = styled(Panel)`
-  padding: 2em;
-`;
+export interface UnntakArsakText {
+  arsak: UnntakArsak;
+  text: string;
+}
+export const unntakArsakTexts: UnntakArsakText[] = [
+  {
+    arsak: UnntakArsak.MEDISINSKE_GRUNNER,
+    text: "Medisinske grunner",
+  },
+  {
+    arsak: UnntakArsak.INNLEGGELSE_INSTITUSJON,
+    text: "Innleggelse i helseinstitusjon",
+  },
+  {
+    arsak: UnntakArsak.FRISKMELDT,
+    text: "Friskmeldt",
+  },
+  {
+    arsak: UnntakArsak.FORVENTET_FRISKMELDING_INNEN_28UKER,
+    text: "Forventet friskmelding innen 28 ukers sykmelding",
+  },
+  {
+    arsak: UnntakArsak.DOKUMENTERT_TILTAK_FRISKMELDING,
+    text: "Tiltak som sannsynligvis vil føre til en friskmelding",
+  },
+  {
+    arsak: UnntakArsak.ARBEIDSFORHOLD_OPPHORT,
+    text: "Arbeidsforholdet er opphørt",
+  },
+];
 
-const SkjemaFieldWrapper = styled.div`
-  margin-bottom: 2em;
-`;
+export const dialogmoteunntakSkjemaBeskrivelseMaxLength = 2000;
+
+const style = {
+  panel: {
+    padding: "2em",
+  },
+  formField: {
+    marginBottom: "2em",
+  },
+  buttonRow: {
+    marginRight: "1em",
+  },
+};
 
```

Comment:
Forslaget om alternativ måte å style på som jeg skriver i PR beskrivelsen

Reply:
Nå går det an å skrive 
`const style = { ghkl: "dsad" }` og passe det inn som style i komponenten. Hvordan sikre type-sikkerhet her slik styled-components gjør?

Reply:
Jeg tror kanskje tailwind-css som mange her på NAV bruker vil kunne løse dette med at man ikke ser stylingen i kontekst av komponenten, så jeg tenker vi heller burde sjekke ut det. Og hvis vi skal fortsatt bruke styled-components; navngi Styled-komponentene bedre.

Reply:
> Jeg tror kanskje tailwind-css som mange her på NAV bruker vil kunne løse dette med at man ikke ser stylingen i kontekst av komponenten, så jeg tenker vi heller burde sjekke ut det. Og hvis vi skal fortsatt bruke styled-components; navngi Styled-komponentene bedre.

Kan godt sjekke ut tailwind😁 Jeg synes at man i det hele tatt må navngi styled komponentene er et problem, siden jeg gjerne vil se direkte hvilken komponent man jobber med som `Panel`, `Button`, o.l og ikke `StyledPanel` eller `SButton` 😊
Tailwind høres ut som det er verdt å sjekke ut synes jeg. Er jo ofte at man egentlig bare trenger 1 `margin-bottom` som fører til at man må lage en styled components og finne på et navn som ikke er forvirrende.

Reply:
> Jeg tror også inline styling er anti-pattern i React, selv om det ikke føles så annerledes ut som når vi bruker styled components 🤔 Se feks [stackoverflow.com/a/60596718](https://stackoverflow.com/a/60596718)

Ser at det ikke er anbefalt ja, så fjerner det 🙌 Ser ut som bruken av det er ment for conditional styling under runtime.

Reply:
> > Jeg tror kanskje tailwind-css som mange her på NAV bruker vil kunne løse dette med at man ikke ser stylingen i kontekst av komponenten, så jeg tenker vi heller burde sjekke ut det. Og hvis vi skal fortsatt bruke styled-components; navngi Styled-komponentene bedre.
> 
> Kan godt sjekke ut tailwind😁 Jeg synes at man i det hele tatt må navngi styled komponentene er et problem, siden jeg gjerne vil se direkte hvilken komponent man jobber med som `Panel`, `Button`, o.l og ikke `StyledPanel` eller `SButton` 😊 Tailwind høres ut som det er verdt å sjekke ut synes jeg. Er jo ofte at man egentlig bare trenger 1 `margin-bottom` som fører til at man må lage en styled components og finne på et navn som ikke er forvirrende.

:+1: Jepp, det er mye sånn margin, spacing-styling vi driver med og mitt inntrykk er at tailwind funker bra til det. Vi får lage en lapp på å teste ut det :smiley: